demo/unified_demo_text_generation/run.py

```diff
@@ -0,0 +1,18 @@
+from transformers import pipeline
```

Comment:
This demo needs a requirements.txt file with transformers and torch in it. Generally a good idea to try uploading the demo to spaces to see if it will work on the website.

Reply:
Thanks Ali for all the comments, I have added the requirement.txt file.